contrib/automation_tests/orbit_capture_loading.py

```diff
@@ -0,0 +1,72 @@
+"""
+Copyright (c) 2021 The Orbit Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+"""
+from absl import app
+
+from core.orbit_e2e import E2ETestSuite
+from test_cases.connection_window import LoadCapture
+from test_cases.live_tab import AddIterator, VerifyFunctionCallCount
+from test_cases.capture_window import CheckTimers, FilterTracks, CheckThreadStates, ToggleCollapsedStateOfAllTracks
+
+"""Verify loading a capture in Orbit using pywinauto.
+
+There is no prerequisites before this script is run.
```

Comment:
At very least you need to have Orbit started, right? And the captures need to be present in the testdata folder.

Reply:
Added this. But note, that this is different in all other test descriptions.

Btw I also added the `hello_ggp_standalone` stanza as I plan to extend this test to also take a capture and click on the first capture in the list of captures afterwards.

Reply:
Changed my mind, only stated orbit for now.